k8s-install/config.yaml

```diff
@@ -0,0 +1,39 @@
+# This ConfigMap can be used to configure a self-hosted Canal installation.
+# See `canal.yaml` for an example of a Canal deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: canal-config 
+  namespace: kube-system
+data:
+  # Configure this with the location of your etcd cluster.
+  etcd_endpoints: "http://127.0.0.1:2379"
+
+  # The interface used by canal for host <-> host communication.
+  canal_iface: "eth1"
+
+  # Whether or not to masquerade traffic to destinations not within
+  # the pod network.
+  masquerade: "true"
+
+  # The CNI network configuration to install on each node.  The special
+  # values in this config will be automatically populated.
```

Comment:
Could do that.

I've got this PR in flight which also documents it (though in the scope of Calico, rather than Canal) https://github.com/projectcalico/calico-containers/pull/1157/files